app/screens/root/root.js

```diff
@@ -29,6 +30,10 @@ export default class Root extends PureComponent {
         }
     }
 
+    componentWillUnmount() {
+        mattermostManaged.clearListeners();
```

Comment:
The issue here is that it won't ever listen for that event cause this screen gets unmounted basically once we switch to the server screen or the channel screen.

we need to figure out another way to clear the listeners